components/tools/OmeroWeb/test/integration/test_plategrid.py

```diff
@@ -419,10 +419,14 @@ def test_get_plate_table(self, django_client, plate_well_table, conn):
         # E.g. webgateway/table/Plate.wells/2061/query/?query=Well-2061
         request_url = reverse('webgateway_object_table_query',
                               args=("Plate.wells", wellId))
-        response = django_client.get(request_url,
-                                     data={'query': 'Well-%s' % wellId})
-        rspJson = json.loads(response.content)
-        print rspJson
+        try:
+            response = django_client.get(request_url,
+                                         data={'query': 'Well-%s' % wellId})
+            rspJson = json.loads(response.content)
+        finally:
+            for v in self.client.getSession().activeServices():
+                assert 'TableProxy' not in v, 'Leaked table!'
```

Comment:
Can't test locally, but are you sure this is ```TableProxy```, not ```TablePrx``` or ```Table```?

Reply:
typo you are correct